src/utils/daos_hdlr.c

```diff
@@ -48,6 +49,37 @@
 
 #include "daos_hdlr.h"
 
+#define NUM_DIRENTS 	24
+#define MAX_FILENAME 	256
+
+struct dfs_daos {
```

Comment:
@daltonbohning rename to fs_copy_dirent?

Reply:
@dsikich That would definitely make it more clear. And I think the main thing is that we shouldn't use the dfs_ prefix because that is reserved for the DFS API.